src/config.cpp

```diff
@@ -835,6 +835,7 @@ namespace config {
     std::vector<std::string> list;
     list_string_f(vars, name, list);
 
+    input.clear();
```

Comment:
Please comment your code, so when someone looks at this in a year or 5 they understand why this is here.

Reply:
The frame rate list must be cleared before adding values from the file configuration. If the list is not cleared, then the specified parameters do not affect the behavior of the sunshine server. That is, if you set only 30 fps in the configuration file, it will not work because by default, during initialization the list includes [10, 30, 60, 90 and 120 fps](https://github.com/LizardByte/Sunshine/blob/31e8b798dabf47d5847a1b485f57cf850a15fcae/src/config.cpp#L423).

Reply:
Thank you for the explanation, but please add it to the code.